platform/jvm/capture/src/main/kotlin/io/bitdrift/capture/Capture.kt

```diff
@@ -106,8 +107,8 @@ object Capture {
         @Suppress("UnusedPrivateMember")
         @ExperimentalBitdriftApi
         @JvmStatic
-        fun initFatalIssueReporting() {
-            val fatalIssueReporter = FatalIssueReporter()
+        fun initFatalIssueReporting(applicationContext: Context) {
```

Comment:
it's weird to call this `applicationContext` since there's no guarantees that that's what the user will be passing (as it will be very well the case when called from content providers)